scripts/asr_language_modeling/ngram_lm/install_beamsearch_decoders.sh

```diff
@@ -1,7 +1,25 @@
 #!/usr/bin/env bash
-# install Boost package
-sudo apt-get update
-sudo apt-get install swig build-essential libboost-all-dev cmake zlib1g-dev libbz2-dev liblzma-dev
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use this script to install KenLM, OpenSeq2Seq decoder, Flashlight decoder
+NEMO_PATH=/workspace/nemo  # Path to NeMo folder: /workspace/nemo if you use NeMo/Dockerfile
```

Comment:
Make Nemo path a bash optional input argument, defaulting to this path.

Reply:
done